Modify seq2seq.py

oxwsds opened the pull request:
Modify the tf.contrib.legacy_seq2seq.embedding_attention_seq2seq so that it allows user to provide a second cell for the encoding step, and if it's None then to fallback on the deepcopy.

oxwsds committed:
Update seq2seq.py

Modify the tf.contrib.legacy_seq2seq.embedding_attention_seq2seq so that it allows user to provide a second cell for the encoding step, and if it's None then to fallback on the deepcopy.

tensorflow-jenkins commented:
Can one of the admins verify this patch?

lukaszkaiser commented on tensorflow/contrib/legacy_seq2seq/python/ops/seq2seq.py:
```diff
@@ -792,6 +792,7 @@ def embedding_attention_seq2seq(encoder_inputs,
                                 num_encoder_symbols,
                                 num_decoder_symbols,
                                 embedding_size,
+                                encoder_cell=None,
```
Please add a description in the "Args:" section of the comment below.

oxwsds replied:
Hi, I have added the description of new arg 'encoder_cell' in comment. Please have a check.

lukaszkaiser reviewed (approved):
Please add to "Args", looks good to merge after that.

oxwsds committed:
Add description of new added arg 'encoder_cell'

Add the description of new added arg 'encoder_cell'  in the 'Args:' section of the comment in tf.contrib.legacy_seq2seq.embedding_attention_seq2seq

lukaszkaiser reviewed (approved):
Thanks, looks good.

caisq commented:
@tensorflow-jenkins test this please

oxwsds committed:
Fix the bad indentation

Fix the bad indentation in tensorflow/contrib/legacy_seq2seq/python/ops/seq2seq.py:854

caisq commented:
@tensorflow-jenkins test this please

drpngx commented:
There was a download error, perhaps transient.

Jenkins, test this please.

drpngx commented:
Git fetch error, trying again.

Jenkins, test this please.

drpngx commented:
BTW, did you mean to send this to `1.1` or `master`?

oxwsds commented:
These changes were based on 1.1, so I wanted to send it to 1.1
            On 04/11/2017 09:05, drpngx wrote: BTW, did you mean to send this to 1.1 or master?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/tensorflow/tensorflow","title":"tensorflow/tensorflow","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/tensorflow/tensorflow"}},"updates":{"snippets":[{"icon":"PERSON","message":"@drpngx in #9037: BTW, did you mean to send this to `1.1` or `master`?"}],"action":{"name":"View Pull Request","url":"https://github.com/tensorflow/tensorflow/pull/9037#issuecomment-293120417"}}}

drpngx commented:
`1.1` is a branch for the TF team to cherry-pick into. If you think that it's a feature that should go into the main tensorflow, you should write into `master`.

/cc: @yifeif

oxwsds commented:
So if I want to do that should I close this PR and create a new PR into master?

yifeif commented:
That would be great @oxwsds.

oxwsds commented:
Well, thank you!

oxwsds commented:
#9122 Please have a check.
